overview.md

```diff
@@ -123,11 +123,11 @@ have the URIs for conformance to actually resolve to machine-readable informatio
 
 | **Name**               | **Specified in**                            | **Conformance URI**                                    | **Description**                                                                                                 |
 | ---------------------- | ------------------------------------------- | ------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------- |
-| STAC API - Core        | [Core](core)                                | <https://api.stacspec.org/v1.0.0-rc.1/core>            | Specifies the STAC Landing page `/`, communicating conformance and available endpoints.                         |
-| STAC API - Browseable  | [Core](core)                                | <https://api.stacspec.org/v1.0.0-rc.1/browseable>      | Advertises .                                                                                                    |
-| STAC API - Item Search | [Item Search](item-search)                  | <https://api.stacspec.org/v1.0.0-rc.1/item-search>     | Enables search of all STAC Item objects on the server, with the STAC `[/search](#stac-api-endpoints)` endpoint. |
-| STAC API - Features    | [Collections and Features](ogcapi-features) | <https://api.stacspec.org/v1.0.0-rc.1/ogcapi-features> | Specifies the use of OGC API - Features to serve STAC Item and Collection objects                               |
-| STAC API - Collections | [Collections and Features](ogcapi-features) | <https://api.stacspec.org/v1.0.0-rc.1/collections>     | Specifies the use of a subset of STAC API - Features to serve Collection objects                                |
+| STAC API - Core        | [Core](core)                                | <https://api.stacspec.org/v1.0.0-rc.2/core>            | Specifies the STAC Landing page `/`, communicating conformance and available endpoints.                         |
+| STAC API - Browseable  | [Core](core)                                | <https://api.stacspec.org/v1.0.0-rc.2/browseable>      | Advertises .                                                                                                    |
```

Comment:
Why does this just say 'Advertises .'?

Reply:
I apparently stopped typing mid-sentence when I added that. I'll create another PR to fix it before release.

Reply:
Created https://github.com/radiantearth/stac-api-spec/pull/346 to fix that.

Reply:
Resolved.